src/etc/inc/plugins.inc.d/openvpn/wizard.inc

```diff
@@ -205,10 +205,9 @@ function step6_stepbeforeformdisplay()
                 break;
             }
         }
-    }
-
-    if ($no_internal_ca) {
+        if ($no_internal_ca) {
```

Comment:
indent looks a bit odd here, if intended, the next line likely needs 4 spaces as well.

Reply:
the if is part of the else now on line 201

Reply:
thanks!